mirgecom/integrators.py

```diff
@@ -46,5 +46,5 @@ def rk4_step(state, t, dt, rhs):
 
 
 def euler_step(state, t, dt, rhs):
-    """Implement a euler explicit time step state/rhs pair."""
+    """Implement an Euler explicit time step state/rhs pair."""
```

Comment:
```suggestion
    """Integrate the ODE given by *rhs* in time by one time step
    using forward Euler.
    """
```
(When you call the function, it doesn't "implement".)